modules/porous_flow/doc/content/source/auxkernels/PorousFlowElementNormal.md

```diff
@@ -0,0 +1,31 @@
+# PorousFlowElementNormal
+
+This `AuxKernel` calculates the element normal, and returns the *x*, *y*, or *z* component.  This is mostly designed for 2D elements living in 3D space, however, the 1D and 3D cases are handled as special cases using the `1D_perp` and `3D_default` inputs.
```

Comment:
I would really suggest using the param syntax, i.e., `[!param](/AuxKernels/PorousFlowElementNormal/1D_perp)` in the future to keep track of parameters in documentation.

Reply:
cool, thanks.  Done